### PR TITLE
Updated indoor-map example to add code for venuesService.getMapInfoLi…

### PR DESCRIPTION
**Pull request by bernardmendonca**
…st()

### CODE REVIEW

File: indoor-map/demo.html [hunk 1]
```diff
@@ -55,9 +55,10 @@ <h1>Show an Indoor Map</h1>
     <div id="map"></div>
     <h3>Code</h3>
     <p>
-      The example shows loading the Indoor Map using <code>H.venues.Service</code> and renders the indoor map using an instance of <code>H.venues.Provider</code>.<br> 
+      The example shows loading the Indoor Map using <code>H.venues.Service2</code> and renders the indoor map using an instance of <code>H.venues.Provider</code>.<br> 
 	    The example also shows how to:
       <ul>
+        <li>get the list of indoor maps from the given HRN using <code>H.venues.Service2.getMapInfoList()</code></li>
         <li>disable the base map while loading a indoor map using <code>H.Map.setBaseLayer (layer)</code></li>
         <li>switch to an indoor map level other than the default using <code>H.venues.Venue.setActiveLevelIndex (levelIndex)</code></li>
```
Comment (dbacarel) [inline indoor-map/demo.html:63]:
```suggestion
        <li>switch to an indoor map level other than the default using <code>H.venues.Venue.setActiveLevelIndex(levelIndex)</code></li>
```

File: indoor-map/demo.html [hunk 2]
```diff
@@ -55,9 +55,10 @@ <h1>Show an Indoor Map</h1>
     <div id="map"></div>
     <h3>Code</h3>
     <p>
-      The example shows loading the Indoor Map using <code>H.venues.Service</code> and renders the indoor map using an instance of <code>H.venues.Provider</code>.<br> 
+      The example shows loading the Indoor Map using <code>H.venues.Service2</code> and renders the indoor map using an instance of <code>H.venues.Provider</code>.<br> 
 	    The example also shows how to:
       <ul>
+        <li>get the list of indoor maps from the given HRN using <code>H.venues.Service2.getMapInfoList()</code></li>
         <li>disable the base map while loading a indoor map using <code>H.Map.setBaseLayer (layer)</code></li>
```
Comment (dbacarel) [inline indoor-map/demo.html:62]:
```suggestion
        <li>disable the base map while loading a indoor map using <code>H.Map.setBaseLayer(layer)</code></li>
```

File: indoor-map/demo.html (hunk 1 above)
Comment (dbacarel) [inline indoor-map/demo.html:63]:
```suggestion
        <li>switch to an indoor map level other than the default using <code>H.venues.Venue.setActiveLevelIndex(levelIndex)</code></li>
```

**Review by dbacarel — CHANGES_REQUESTED** [review]
typos only to be fixed

File: indoor-map/demo.html (hunk 1 above)
Comment (bernardmendonca) [reply indoor-map/demo.html]: Done

File: indoor-map/demo.html (hunk 2 above)
Comment (bernardmendonca) [reply indoor-map/demo.html]: Done

File: indoor-map/demo.html (hunk 1 above)
Comment (bernardmendonca) [reply indoor-map/demo.html]: Done. This is same as the first comment right?

**Review by dbacarel — APPROVED** [review]

**Pull request merged**